.github/workflows/build-gst_capi.yaml

```diff
@@ -0,0 +1,30 @@
+name: Build docker images for GST C-API
+run-name: GST C-API image builder
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - use-cases/gst_capi/**
```

Comment:
the c-api could take very long to build, maybe change it to using nightly cron-job schedule to save some waiting time while we are sleeping :-)

Reply:
Yeah, usually around 35-40 minutes...
Okay, good point. 
I was thinking about doing that with this one...